auto3dseg/notebooks/msd_datalist_generator.ipynb

```diff
@@ -0,0 +1,495 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "Copyright (c) MONAI Consortium\n",
```

Comment:
To avoid the lines collapse in the notebook, please add two extra space in the end. For example:
The first line is
"Copyright (c) MONAI Consortium\n",
and it should be
"Copyright (c) MONAI Consortium  \n",

The last line is
"limitations under the License.\n",
and it should be
"limitations under the License.  \n",

Reply:
Maybe I confused you with my comment and some auto-correction in the code editor. Can you replace the copyright by copy and pasting the following snippet?

```
Copyright (c) MONAI Consortium  
Licensed under the Apache License, Version 2.0 (the "License");  
you may not use this file except in compliance with the License.  
You may obtain a copy of the License at  
    http://www.apache.org/licenses/LICENSE-2.0  
Unless required by applicable law or agreed to in writing, software  
distributed under the License is distributed on an "AS IS" BASIS,  
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  
See the License for the specific language governing permissions and  
limitations under the License.  
```

Reply:
Hi @udiram , the blank spaces disappear. It may be communication or auto-correction from the code editor. Can you copy paste this snippet in the notebook to replace your header? Thanks!

```
Copyright (c) MONAI Consortium  
Licensed under the Apache License, Version 2.0 (the "License");  
you may not use this file except in compliance with the License.  
You may obtain a copy of the License at  
    http://www.apache.org/licenses/LICENSE-2.0  
Unless required by applicable law or agreed to in writing, software  
distributed under the License is distributed on an "AS IS" BASIS,  
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  
See the License for the specific language governing permissions and  
limitations under the License.  
```